client/register/register.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+<h3>Cadastro</h3>
+<form *ngIf="active" #f="ngForm" novalidate (ngSubmit)="onSubmit()">
+  <div class="form-group row {{name.pristine?'':(name.valid?'has-success':'has-danger')}}">
+    <label class="col-sm-2 col-form-label">Nome</label>
+    <div class="col-sm-10">
+      <input type="text" class="form-control {{name.valid || name.pristine?'form-control-success':'form-control-danger'}}" name="name" id="name" placeholder="Nome" required #name="ngModel" [(ngModel)]="model.name">
+      <div [hidden]="name.valid || name.pristine" class="form-control-feedback">
+        Nome não pode ser vazio
+      </div>
+    </div>
+  </div>
+  <div class="form-group row {{email.pristine?'':(email.valid?'has-success':'has-danger')}}">
+    <label class="col-sm-2 col-form-label">Email</label>
+    <div class="col-sm-10">
+      <input type="email" class="form-control {{email.valid || email.pristine?'form-control-success':'form-control-danger'}}" name="email" id="email" placeholder="Email" validateEmail required pattern="^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+$" #email="ngModel" [(ngModel)]="model.email">
```

Comment:
O e-mail é válido se respeitar a expressão regular (contiver @ e/ou .), e se já não for cadastrado no servidor (validateEmail), que é o nosso EmailExistingValidator